README.md

```diff
@@ -1 +1,115 @@
 
+# Farnaz Mozaffari . Jarvis Consulting
+I recently completed my master's degree in Computer Engineering from University of Guelph with a focus on AI applications. 
+During my master's studies, I worked with different datasets and developed machine learning and deep learning models mainly in python for various applications such as cybersecurity systems. 
+With my strong background in analyzing and interpreting large dataset and development of highly accurate forecasting systems, I started my training program at Jarvis Consulting group. 
+Developing various projects at Jarvis led me to become skilled in debugging and increasing code efficiency in PostgreSQL and Java Stream APIs as well as gaining comprehensive knowledge of REST APIs, and Spring framework. 
+<br /> I am a self-initiated and motivated person with a passion in implementing complex algorithms. Also, I am eager to have positive impact in business transformation and performance improvement. 
+
+With my master of Computer Engineering degree and rewarding experience at Jarvis, I am confident I will be a great candidate for Software Development opportunities mainly in the fields of Data and Machine-learning Engineering.
+
+
+## Skills
+
+**Proficient:** Java, Python, Linux/Bash, SQL, Git, Machine
+ Learning , Deep Learning , Agile/Scrum
```

Comment:
fix spaces before commas 

be more specific on Machine learning and Deep learning.
e.g. Deep Learning (CNN, other algorithms or your focus area)